kirby/api/ext.py

```diff
@@ -1,61 +1,145 @@
+import logging
+import datetime
 import msgpack
 from kafka import KafkaConsumer, KafkaProducer
+from kafka.errors import NoBrokersAvailable
+
+logger = logging.getLogger(__name__)
+
+
+def retry(function):
+
+    NB_RETRY = 3
```

Comment:
Why `3`, and why not getting it from the environment? 

Can you move it to the module level please?

Reply:
Now get from the environment variables.